packages/debugger-extension/src/index.ts

```diff
@@ -721,19 +721,21 @@ const main: JupyterFrontEndPlugin<void> = {
     });
 
     commands.addCommand(CommandIDs.pause, {
-      label: service.isPausingOnExceptions
-        ? trans.__('Disable pausing on exceptions')
-        : trans.__('Enable pausing on exceptions'),
-      caption: trans.__('Enable / Disable pausing on exceptions'),
+      label: trans.__('Enable / Disable pausing on exceptions'),
+      caption: () =>
+        service.isStarted
+          ? service.pauseOnExceptionsIsValid()
+            ? service.isPausingOnExceptions
+              ? trans.__('Disable pausing on exceptions')
+              : trans.__('Enable pausing on exceptions')
+            : trans.__('Kernel does not support pausing on exceptions.')
+          : trans.__('Enable / Disable pausing on exceptions'),
```

Comment:
I would drop the `service.isStarted` to focus on providing the info if the kernel does not support the feature or the button state.

```suggestion
        service.pauseOnExceptionsIsValid()
          ? service.isPausingOnExceptions
            ? trans.__('Disable pausing on exceptions')
            : trans.__('Enable pausing on exceptions')
          : trans.__('Kernel does not support pausing on exceptions.'),
```

Reply:
I added in the `service.isStarted` logic because otherwise, if someone hovers over the button when the debugger is turned off, it will show the the kernel does not support pausing on exceptions which may not be true and could be a misdirection in case they turn on the debugger and don't hover again to see that the kernel actually does support it. I'm happy to change this, but thought it was worth mentioning. I went back and forth on it a few times when I wrote this part.